UnityProject/Assets/Scripts/MapSaver/MapSaver.cs

```diff
@@ -372,23 +372,23 @@ public int Compare(string x, string y)
 							z2 = float.Parse(partsY[2]);
 						}
 
-						if (x1 != x2)
+						if (Mathf.Approximately(x1, x2) == false)
```

Comment:
you shouldn't need to use Mathf.Approximately, Since the coordinates are rounded, this is just a sorting function

Reply:
well They are rounded to n number if a member 3 decimal places it might go up to 6 if it's needed